src/index.js

```diff
@@ -40,36 +41,49 @@ export { default as Utils } from './utils';
  * - MDN: [HTMLTemplateElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement)
  */
 
-/*
- * Register element definitions with the Custom Element registry.
- */
-function _defineElements () {
-    for (let element in Elements) {
-        Elements[element].$define();
-    }
-}
+const HelixUI = {
+    Elements,
+    Utils,
+    VERSION,
```

Comment:
We might be able to retain the named export style and convert to a default export in the entry points. This should allow for better flexibility.  Need to verify that if a `config` export were defined that internal workings can still reference user-defined configurations from within.